lib/find.js

```diff
@@ -30,7 +30,7 @@ function pointFinder(name) {
 
             return data.Points.filter(function (p) {
                 p = p.split('|');
-                return p[2].length === 0 || p[2] === 'Dresden';
+                return p[3] && (p[2].length === 0 || p[2] === 'Dresden');
```

Comment:
I identified this by one of the test cases which were failing because it used exactly this `find("0",...)` syntax.